src/components/entry/EntryDetails.js

```diff
@@ -151,6 +180,21 @@ const EntryDetails = () => {
         {typesHeader}
         <Description>
           <p>{locationData.description}</p>
+          <IconBesideText bold onClick={_handleAddressClick} tabIndex={0}>
+            <Map color={theme.secondaryText} size={20} />
+            <LocationText>{address}</LocationText>
+          </IconBesideText>
+          {locationData.season_start && locationData.season_stop && (
```

Comment:
Just checking, did you find any examples of no_season and how that case should be displayed?